javascript/javascript3/week1/GiphyApi.js

```diff
@@ -0,0 +1,42 @@
+
+const search = document.getElementById('searchText')
+const resultNumber = document.getElementById('number')
+const button = document.querySelector('button')
+const div = document.getElementById('center')
+
+search.addEventListener('change',startSearch)
+
+
+   function startSearch(){
+      if (search.value){
+      div.innerHTML = 'Please enter number to start search or just press button!'
+      button.addEventListener('click',searchButton)
+
+   }
+      else{
+         div.innerHTML = 'Please fill the search term!'
+      }
+   }
+
+            function searchButton(){
+            div.innerHTML = '';
+               if (resultNumber.value!='Number of gifs'){
+               fetchGif(search.value,resultNumber.value)
+            }
+               else {
+                  fetchGif(search.value,4)
+               }
+            }
+
+                           function fetchGif(searchedText,result){
```

Comment:
I am not sure if it is necessary to give indent for functions. you can start defining your function from the beginning of your line, not the middle.